tests/e2e/specs/LicenseCode.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
 
```

Comment:
I would like to see some tests in here that check to verify that the HTML, richtext, and print attributions are being generated properly and have the correct values. For example, if CC BY-NC is selected, I'd like to see a check that the print attribution is of the value `This work is licensed under CC BY-NC 4.0. To view a copy of this license, visit https://creativecommons.org/licenses/by-nc/4.0`.

Reply:
Ya, I added them, there were many span tags so I had to add a test for each of them... but done:)